src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Right now some of your tests are failing. You'll be equally required to fix them.
![Screenshot from 2023-05-25 20-07-07](https://github.com/zunairkhan811/react-stocks-project/assets/63087491/c5d706bb-4749-44f7-89c0-31d907612041)

Reply:
Added more tests and test are working fine now